src/openfhe.jl

```diff
@@ -38,9 +38,9 @@ end
 function init_bootstrapping!(context::SecureContext{<:OpenFHEBackend},
                              private_key::PrivateKey)
     cc = get_crypto_context(context)
-    ring_dimension = OpenFHE.GetRingDimension(cc)
-    num_slots = div(ring_dimension,  2)
-    OpenFHE.EvalBootstrapKeyGen(cc, private_key.private_key, num_slots)
+    encoding_parameters = OpenFHE.GetEncodingParams(cc)
+    capacity = OpenFHE.GetBatchSize(encoding_parameters)
+    OpenFHE.EvalBootstrapKeyGen(cc, private_key.private_key, capacity)
```

Comment:
```suggestion
    slots = OpenFHE.GetBatchSize(encoding_parameters)
    OpenFHE.EvalBootstrapKeyGen(cc, private_key.private_key, slots)
```
Since we rely on OpenFHE internals here, I'd just call it how OpenFHE does ("slots")